src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -5,10 +5,14 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Http
 {
     public class HttpWorkerOptions
     {
+        public string Type { get; set; } = "http";
```

Comment:
@anthonychu - should we leave out the `Type`? We can introduce `Type`, in future if we ever decide to expand customHandlers to any other tranport protocol. We can then default to `http` if type is not sepcified.

Reply:
We can leave this  out of documentation. Defaults to `http` for now.